astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+# Originally authored by Jazmin Berlanga Medina
+# (jazmin.berlanga@gmail.com), with input from
+# Adrian Price-Whelan (email), Erik Tollerud (email),
+# Christoph Deil (email), Eric Jeschke (email) and Brett Morris (email).
+
+
+# Replace function call later with below.
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None, dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) Airmass equation? Should this be moved to a separate function?
+        6) dark_plot option
+
+    Returns an ax object with an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        An collection of times to use in generating data to plot.
+        Can be scalar (len(time)=1) or contain multiple times.
+
+    ax : `~matplotlib.axes.Axes` or None
+        Optional.
+
+    style_kwargs : WHAT TYPE IS DICTIONARY? or None
+        Optional.
+        A dictionary with plotting style options.
+
+    dark_plot: Boolean.
+        Optional. False (default) or True.
+    """
+
+    # Set up plot axes and style if needed.
+    if ax is None:
+        ax = plt.gca()
+    if style_kwargs is None:
+        # Need to add more style options for ax's with existing plots?
+        style_kwargs = {'linestyle': '-', 'color': 'r'}
```

Comment:
There's a slightly cleaner way of dealing with `style_kwargs`.  Something like this:

```python
style_kwargs = dict(style_kwargs) # make a copy so we don't overwrite what the user gives us in case they want it
style_kwargs.setdefault('linestyle', '-')
style_kwargs.setdefault('color', 'r')
```
That way if the user sets color but not linestyle, they still get the settings you have as defaults (or vice versa).

Note that in this approach the default of `None` is incalid, but an empty dict will do just fine, so you use the empty dict as the default.